installer/conf/omi_mapping.json

```diff
@@ -40,6 +40,46 @@
         ]
     },
     {
+        "CimClassName": "SCX_EthernetPortStatistics",
+        "ObjectName": "Network",
+        "InstanceProperty": "InstanceID",
+        "Namespace": "root/scx",
+        "CimProperties": [
+            {
+                "CimPropertyName": "BytesTransmitted",
+                "CounterName": "Total Bytes Transmitted"
```

Comment:
@agup006, which period does the "Total" refer to? from the last restart? from the last query?